usaspending_api/etl/award_helpers.py

```diff
@@ -41,6 +41,7 @@ def update_awards(award_tuple=None):
         'txn_totals AS ('
         'SELECT award_id, SUM(federal_action_obligation) AS total_obligation, '
         'SUM(original_loan_subsidy_cost) AS total_subsidy_cost, '
+        'SUM(NULLIF(tn.funding_amount, '')::decimal) AS total_funding_amount, '
```

Comment:
If the corresponding model updates above are made to change the text fields to decimal fields, the `NULLIF` and cast to `decimal` here would no longer be needed.